src/orders/components/OrderRefundReturnAmount/utils.ts

```diff
@@ -80,11 +114,33 @@ export const getProductsAmountValues = (
   };
 };
 
+const getReturnTotalAmount = ({
+  selectedProductsValue,
+  refundShipmentCosts,
+  order
+}: {
+  order: OrderDetails_order;
+  selectedProductsValue: IMoney;
+  refundShipmentCosts: boolean;
+}) => {
+  if (refundShipmentCosts) {
+    return selectedProductsValue?.amount + getShipmentCost(order)?.amount;
```

Comment:
Possible `NaN` value, if `undefined` is a possible return value of `getReturnTotalAmount` fn I'd propose to create a fallback here.